MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDrawable.java

```diff
@@ -337,7 +337,7 @@ private void drawShadow(Canvas canvas) {
      */
     private void update(boolean force, boolean async) {
         View view = mViewRef.get();
-        if (view != null) {
```

Comment:
I see 😄  I am also unsure about the best course of action. Let's keep it this way for now 👍